src/Model/List/ArrayList.php

```diff
@@ -585,19 +560,17 @@ public function canFilterBy($by)
     /**
      * Find the first item of this list where the given key = value
      *
-     * @param string $key
-     * @param mixed $value
      * @return T|null
      */
-    public function find($key, $value)
+    public function find(string $key, mixed $value): mixed
```

Comment:
It is _theoretically_ possible to use `ints` as keys in `ArrayList` - but I'm okay with explicitly disallowing that. It _will_ need to be called out in the changelog though.

Reply:
Have updated changelog PR